src/js/index.tsx

```diff
@@ -173,7 +179,7 @@ init_i18n((i18nError?: string) => {
             >
               <App />
             </IconSettings>
-          </BrowserRouter>
+          </ConnectedRouter>
```

Comment:
What's this for?

Reply:
@davisagli This is part of https://github.com/supasate/connected-react-router, which exposes the history/router to our Redux state as well as our components. See https://github.com/oddbird/MetaDeploy/blob/run-job/src/js/store/jobs/actions.ts#L161-L174